sdk/storage/azure-storage-blob/azure/storage/blob/container_client.py

```diff
@@ -655,6 +665,7 @@ def set_container_access_policy(
         except StorageErrorException as error:
             process_storage_error(error)
 
+    @distributed_trace
     def list_blobs(self, name_starts_with=None, include=None, marker=None, timeout=None, **kwargs):
```

Comment:
I'm guessing you have already discussed with Laurent regarding his updates to the paging implementation and how this interacts with tracing? Is anything required on the paging models themselves?

Reply:
So for paging, the agreement was that the list_* will get the span and each network call will be traced. The function that does the individual network call for paging will not be traced as of preview 2.